### PR TITLE
Case study added as example

### PR DESCRIPTION
**Pull request by garvk**
This adds examples/ directory, and adds metadata extraction as first case study. + minor changes to .env.sample and 'viewport' setting of cforge file.

### CODE REVIEW

**Comment by garvk** [comment]
This adds examples/ directory, and adds metadata extraction as first case study. + minor changes to .env.sample and 'viewport' setting of cforge file.

File: examples/metadata_extraction/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+In this example, we are evaluating generated responses' strict adherence to metadata schema and the time taken to extract. 
+
+Note: ideally, adding more than one paper should not affect the time taken as aggrag library is designed to work asynchronously. But in the current example iterations, we are noticing some latency being added as more pdf papers are added. 
+
+## Testing the Flow
+
+You can get your hands dirty by setting up an example flow to learn the capabilities of what  the aggrag example flows by following these steps:
+
+1. Setup and run the aggrag project locally. Steps here: `https://github.com/garvk/aggrag#local-setup`
```
Comment (tarun-goyal) [inline examples/metadata_extraction/README.md]: Ideally, the link `https://github.com/garvk/aggrag#local-setup` should be from the main aggrag repo. Currently, it is from your forked repo.

File: examples/metadata_extraction/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+2. Create a new 'use-case' on aggrag UI.
+3. On iteration 1, import a sample cforge file located in examples/ directory. 'Import' is button on top-right banner on the UI.
+4. To run the flow locally, you'll need to set up the aggrag environment and provide your model API keys in the .env file. Follow the instructions in the Local Setup section of the README.
+
```
Comment (tarun-goyal) [inline examples/metadata_extraction/README.md:34]: Should we also mention that one can follow the provided "hints" to create a new iteration.

File: examples/metadata_extraction/README.md (hunk 1 above)
Comment (garvk) [reply examples/metadata_extraction/README.md]: Yes, you're right. It's my mistake.

File: examples/metadata_extraction/README.md (hunk 2 above)
Comment (garvk) [reply examples/metadata_extraction/README.md:34]: Yes, adding.

File: examples/metadata_extraction/README.md (hunk 1 above)
Comment (tarun-goyal) [reply examples/metadata_extraction/README.md]: @garvk The forked repo link is still here. I don't see any changes.

**Review by tarun-goyal — APPROVED** [review]

**Pull request merged**